app/src/pages/playground/schemas.ts

```diff
@@ -0,0 +1,76 @@
+import { z } from "zod";
+
+import {
+  ImageAttributesPostfixes,
+  LLMAttributePostfixes,
+  MessageAttributePostfixes,
+  MessageContentsAttributePostfixes,
+  SemanticAttributePrefixes,
+} from "@arizeai/openinference-semantic-conventions";
+
+import { CHAT_MESSAGE_ROLES } from "@phoenix/store";
+
+/**
+ * The zod schema for llm tool calls in an input message
+ * @see {@link https://github.com/Arize-ai/openinference/blob/main/spec/semantic_conventions.md|Semantic Conventions}
+ */
+const toolCallSchema = z
+  .object({
+    function: z
+      .object({
+        name: z.string(),
+        arguments: z.string(),
+      })
+      .partial(),
+  })
+  .partial();
+
+/**
+ * The zod schema for llm message contents
+ * @see {@link https://github.com/Arize-ai/openinference/blob/main/spec/semantic_conventions.md|Semantic Conventions}
+ */
+const messageContentSchema = z.object({
```

Comment:
these are currently unused, but will be necessary in the future so i didn't want to remove them, can remove for now and add back later.